app/views/users/show.html.haml

```diff
@@ -1,25 +1,10 @@
-.wrapper
-  .side-bar.d-flex.flex-column.align-items-center
-    = image_tag current_user.avatar, class: 'rounded-circle user-img'
-    .data
-      = current_user.nickname
-      %br
-      = current_user.email
-      %br
-      = current_user.first_name
-      = current_user.last_name
-      %br
-      = current_user.birthday.strftime('%d/%m/%Y')
-      %br
-      %br
-      = 'Posts: '
-      = @posts.count
-      %br
-      = link_to 'Edit', edit_user_registration_path, class: 'btn btn-primary btn-newPost'
+- if @user && @user.nickname.present?
```

Comment:
This line is also unnecessary. If user doesn’t exist, the controller will redirect into root page.